pages/projects/[projectId]/versions/[versionId]/index.tsx

```diff
@@ -236,7 +243,13 @@ export default function VersionPage({
         <Row>
           <Col>
             <Chart data={data} sortedReleases={sortedReleases}></Chart>
-            <Spacer y={3} />
+            <Spacer y={1} />
+            {data.length - positiveChartPoints.length > 0 && (
+              <Text style={{color: 'red'}}>
+                {`You have completed more ${humaniseUnit(project.unit)} than expected this sprint, please edit the volume`}
```

Comment:
`You have completed more ${project.unit.toLocaleLowerCase()}s) that expected this sprint, please edit the volume`